share/bash-completion/completions/mech-dump

```diff
@@ -0,0 +1,12 @@
+_comp_cmd_mech-dump()
```

Comment:
The function name containing `-` fails when the POSIX mode is turned on.

```suggestion
_comp_cmd_mech_dump()
```

Reply:
Good thing you caught that!
I simply typed wrong without noticing.
I wonder if there is any static checker to prevent these kinds of issues.